src/pulseaudio_dbus/init.lua

```diff
@@ -349,6 +349,24 @@ function pulse.Device:toggle_muted()
   return self:is_muted()
 end
 
+--- Get the current active port object path
+-- @return the active port object path
+-- @return nil if no active port is set
+-- @see pulse.Device:set_active_port
+function pulse.Device:get_active_port()
+  return self:Get("org.PulseAudio.Device", "ActivePort")
+end
+
+--- Set the active port object path
+-- @tparam string value port object path
+-- @see pulse.Device:get_active_port
+function pulse.Device:set_active_port(value)
+  self:Set("org.PulseAudio.Core1.Device",
```

Comment:
Since `value` must be a valid port, can you please add a check using the `Ports` property to ensure that `value` is correct please?

Something like

``` lua
local available_ports = self:Get("org.PulseAudio.Core1.Device", "Ports")
-- assert that value is in available_ports
self:Set(...)
```